.changeset/cli-dev-shim.md

```diff
@@ -0,0 +1,7 @@
+---
+'@crackle/cli': minor
+---
+
+Add a `crackle dev --shim` option with choices `none` and `require` (default)
+
+The `crackle dev --webpack` option has been removed, replaced with `crackle dev --shim=none`
```

Comment:
Given this is a breaking change should maybe make this more visible. I understand we are pre 1.0, but maybe something like:
````suggestion
### BREAKING CHANGE:
The `--webpack` option has been removed from the `dev` command in favour of the more generic `--shim` option. 

Consumers should migrate to use `--shim=none` as follows:
```diff
- crackle dev --webpack
+ crackle dev --shim=none
```
````

Reply:
We're still on `0.x`, so I don't think it's worth drawing attention to the breaking change. I will update the wording 👍